htsget-config/src/config.rs

```diff
@@ -67,6 +75,10 @@ pub struct Config {
   pub htsget_updated_at: Option<String>,
   pub htsget_environment: Option<String>,
   pub htsget_s3_bucket: Option<String>,
+  pub htsget_localstorage_ip: String,
+  pub htsget_localstorage_port: String,
```

Comment:
@mmalenic Reviewing this, not sure it's ready to merge but I just noticed that ports/ips still don't have the more fitting networking types?

Reply:
I was planning on changing those in the next PR, where I edit some of the config stuff. For example, a user should be able to choose storage type (LocalStorage vs S3 Storage), which is not yet possible. That should be a fairly small PR.

Reply:
Gotcha, looks like this PR doesn't break anything, so I would keep merging otherwise things pile up! ;)